code/pizzabot.js

```diff
@@ -1,7 +1,89 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
 
-const pizzaPrice = 80
+const pizzaPrice = 80;
+
+let botSays = document.getElementById('botSays');
+botSays.innerHTML = 'What would you like to order?';
+
+// make inputbox actice (the mouse/cursor is there in beginning)
+document.getElementById("answer").focus();
+
+// used to check if valid pizza is ordered
+let orderedPizza = "";
+
+const isValidPizza = (pizza) => {
+    if (vegetarian.toLowerCase() === pizza.toLowerCase() ||
+        hawaiian.toLowerCase() === pizza.toLowerCase() ||
+        pepperoni.toLowerCase() === pizza.toLowerCase()) {
+        return true
+    } else {
+        return false
+    }
+}
+
+const calculateTotalCost = (orderQuantity, pizzaPrice) => {
+    let totalCost = orderQuantity * pizzaPrice
+    return totalCost
```

Comment:
We can simplify this function. Since you are creating `let` variable (btw it should be `const`, because you are not overwriting its value anywhere) and returning it very next line, we could declare it like this
```
const calculateTotalCost = (orderQuantity, pizzaPrice) => {
    return orderQuantity * pizzaPrice;
}
```
Another tip from my side - since we have only one line inside `calculateTotalCost` function, we can write it in "arrow function shorthand" :

`const calculateTotalCost = (orderQuantity, pizzaPrice) => orderQty * pizzaPrice;`

We can do that because in arrow functions we can omit curly brackets (`{}`) and `return` keyword if we only have one line of code inside. At the end, this snippet of code is exactly the same as the first one I wrote above.

For further reference on that subject, please check out https://www.w3schools.com/js/js_arrow_function.asp